xcp_d/utils/bids.py

```diff
@@ -272,22 +272,22 @@ def collect_data(
     elif t1w_files and t2w_files:
         LOGGER.warning('Both T1w and T2w found. Checking for T1w-space T2w.')
         queries['t2w']['space'] = 'T1w'
-        t2w_found = bool(
+        t1wspace_t2w_found = bool(
             layout.get(return_type='file', subject=participant_label, **queries['t2w'])
         )
-        if not t2w_found:
+        if not t1wspace_t2w_found:
             LOGGER.warning('No T1w-space T2w found. Checking for T2w-space T1w.')
             queries['t1w']['space'] = 'T2w'
             queries['t2w']['space'] = None  # we want T2w without space entity
-            t1w_found = bool(
+            t2wspace_t1w_found = bool(
                 layout.get(
                     return_type='file',
                     subject=participant_label,
                     **queries['t1w'],
                 )
             )
-            if not t1w_found:
-                LOGGER.warning('No T2w-space T1w found. Attempting T2w-only processing.')
+            if not t2wspace_t1w_found:
```

Comment:
I get confused because there is a T2-SPACE acquisition. This is referring to the bids entity, right?

Reply:
Oh, as opposed to the sequence? Yeah it's the BIDS entity- so we're looking for `space-T2w_T1w` files, which could be created if the pipeline coregistered the secondary anatomical contrast to the primary anatomical contrast before running Freesurfer and normalization, AFAICT. I don't think fMRIPrep or Nibabies actually does that yet though, so this is more future-proofing.